staging/src/k8s.io/apimachinery/pkg/util/validation/validation_test.go

```diff
@@ -238,7 +239,7 @@ func TestIsQualifiedName(t *testing.T) {
 		"example.com/Uppercase_Is_OK_123",
 		"requests.storage-foo",
 		strings.Repeat("a", 63),
-		strings.Repeat("a", 253) + "/" + strings.Repeat("b", 63),
```

Comment:
Do we have concrete cases of individual parts being > 63?  I guess even if not, this is scary at this point.